config/game_options.txt

```diff
@@ -492,4 +492,4 @@ MICE_ROUNDSTART 10
 #EMERGENCY_SHUTTLE_AUTOCALL_THRESHOLD 0.2
 
 ## Determines if players are allowed to print integrated circuits, uncomment to allow.
-#IC_PRINTING
\ No newline at end of file
+#IC_PRINTING
```

Comment:
undocumented change

Reply:
it hasn't changed. the last line is always green-red in github changes.

```
-#IC_PRINTING
+#IC_PRINTING
```

literally nothing changes

Reply:
line endings etc